Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -60,6 +61,11 @@
 leader_id_number = 999999099
 leader_grant_payload = { 'username':leader_username, 'password':leader_password, 'grant_type':'password' }
 
+location = 'KOSC 244'
```

Comment:
Maybe it would be better if you put a comment around this to specify what it is used for.

Reply:
Added